src/cluster/replication.cc

```diff
@@ -1000,15 +1000,15 @@ Status ReplicationThread::parseWriteBatch(const std::string &batch_string) {
 }
 
 bool ReplicationThread::isRestoringError(std::string_view err) {
-  return err == std::string(RESP_PREFIX_ERROR) + redis::errRestoringBackup;
+  return err == redis::Error({Status::RedisLoading, redis::errRestoringBackup});
```

Comment:
```suggestion
  return err == std::string(RESP_PREFIX_ERROR) + redis::StatusToRedisError({Status::RedisLoading, redis::errRestoringBackup});
```

Reply:
It's the same with Redis::Error

Reply:
Ahh, let's remove CRLF in StatusToRedisError.

Reply:
CRLF should not be in StatusToRedisError. It will make the semantics of StatusToRedisError very weird.

Reply:
done